src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.Integer.cs

```diff
@@ -102,27 +105,63 @@ private static ulong ReadUnsignedInteger(ReadOnlySpan<byte> buffer, CborInitialB
 
                 case CborAdditionalInfo.Additional8BitData:
                     EnsureBuffer(buffer, 2);
+                    result = buffer[1];
+
+                    if (result < (int)CborAdditionalInfo.Additional8BitData)
```

Comment:
Since the pattern changes between this case label (which verifies against itself) and the next (which verifies against byte.MaxValue) it might be worth a comment (or another const with a value of `Additional8BitData - 1`) reminding readers of how low values are/can-be carried in the initial byte.